temperatures/routes.py

```diff
@@ -0,0 +1,38 @@
+from datetime import datetime
+from typing import List, Optional
+
+from fastapi import Depends, HTTPException
+from sqlalchemy.orm import Session
+
+from cities.models import DbCity
+from dependencies import app, get_db, fetch_temperature, ListDep
+from temperatures import crud
+from temperatures.models import DbTemperature
+from temperatures.schemas import Temperature
+
+
+@app.post("/temperatures/update")
+async def update_temperatures(db: Session = Depends(get_db)) -> dict[str, str]:
```

Comment:
The return type hint `dict[str, str]` is not compatible with Python versions below 3.9. If you need to support older versions, consider using `Dict[str, str]` from the `typing` module.